### PR TITLE
Canonicalize the attribute list when printing a `LazyTensorOperation`

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 1]
```diff
@@ -682,7 +682,7 @@ extension LazyTensor: CustomStringConvertible {
 
 extension LazyTensorOperation: CustomStringConvertible {
     public var description: String {
-        let attrsDesc = attrs.map { (name, value) in "\(name): \(value)" }
+        let attrsDesc = attrs.keys.sorted(by: <).map { "\($0): \(attrs[$0]!)" }
```
Comment (dan-zheng) [inline Sources/TensorFlow/Core/LazyTensorOperation.swift]: Please use [`Dictionary.sorted(by:)`](https://developer.apple.com/documentation/swift/dictionary/2298238-sorted).
```suggestion
        let attrsDesc = attrs.sorted(by: { $0.key < $1.key }).map { "\($0): \($1)" }
```

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 2]
```diff
@@ -682,7 +682,7 @@ extension LazyTensor: CustomStringConvertible {
 
 extension LazyTensorOperation: CustomStringConvertible {
     public var description: String {
-        let attrsDesc = attrs.map { (name, value) in "\(name): \(value)" }
+        let attrsDesc = attrs.sorted(by: { $0.key < $1.key }).map { "\($0): \($1)" }
```
[the remark this answers is not in the document]
Comment (bgogul) [reply Sources/TensorFlow/Core/LazyTensorOperation.swift]: Let me send out a follow up PR for this.

File: Sources/TensorFlow/Core/LazyTensorOperation.swift [hunk 3]
```diff
@@ -682,7 +682,7 @@ extension LazyTensor: CustomStringConvertible {
...
         let inputsDesc = inputs.map { (input: Input) -> String in
             switch input {
             case Input.single(let lazyTensor):
```
[the remark this answers is not in the document]
Comment (bgogul) [reply Sources/TensorFlow/Core/LazyTensorOperation.swift]: Done.

[the remark this answers is not in the document]
Comment (bgogul) [reply Sources/TensorFlow/Core/LazyTensorOperation.swift]: Done.

**Pull request merged**